api/blueprint/templates_api.go

```diff
@@ -89,7 +89,7 @@ func (tp *TemplateService) CreateTemplate(templateVector *map[string]interface{}
 func (tp *TemplateService) UpdateTemplate(templateVector *map[string]interface{}, ID string) (template *types.Template, err error) {
```

Comment:
We are going to need a new template compile command that makes use of the new 'PUT /v2/templates/:id/compile' resource

Reply:
Done